OracleWebLogic/dockerfiles/14.1.1.0/Dockerfile.slim-8

```diff
@@ -0,0 +1,110 @@
+#Copyright (c) 2014-2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
There is a delay in issuing a 14.1.1.0 slim, correct?

Reply:
Correct the Slim installer is not available at this moment to create images.